script/gen_requirements_all.py

```diff
@@ -165,6 +165,7 @@
     "sleepyq",
     "smhi-pkg",
     "solaredge",
+    "sunwatcher",
```

Comment:
Don't add it here, this is only for integrations that have tests.

Reply:
Removed